<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(58, 97, 134, 255), stop:1 rgba(137, 37, 62, 255));

background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #bdc3c7, stop:1 #2c3e50);

background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.99, y2:0.994318, stop:0 rgba(1, 114, 145, 255), stop:0.58209 rgba(25, 39, 59, 255));

background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.99, y2:0.994318, stop:0 rgba(57, 109, 121, 255), stop:1 rgba(25, 39, 59, 255));</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>11</number>
   </property>
   <property name="topMargin">
    <number>11</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
background: none;
border-radius: 20px;

background-color: rgb(47, 72, 99);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(20, 30, 48, 255), stop:1 rgba(36, 59, 85, 255));
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0" alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="home">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{  
width: 30%;
 color: #fff;
  padding: 5px;
  font-size: 18px;
  text-transform: uppercase;
  border-radius: 10px;
border:none;
background: none;

}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(40, 0, 114, 255), stop:1 rgba(46, 191, 145, 255));
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;
background:none;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{  
width: 30%;
 color: #fff;
  padding: 5px;
  font-size: 18px;
  text-transform: uppercase;
  border-radius: 10px;
border:none;
background: none;
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(40, 0, 114, 255), stop:1 rgba(46, 191, 145, 255));
}
QPushButton:pressed{
background-color:qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(50, 0, 143, 255), stop:1 rgba(39, 163, 122, 255))
}</string>
        </property>
        <property name="text">
         <string>View</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="upload">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{  
width: 30%;
 color: #fff;
  padding: 5px;
  font-size: 18px;
  text-transform: uppercase;
  border-radius: 10px;
border:none;
background: none;
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(40, 0, 114, 255), stop:1 rgba(46, 191, 145, 255));
}
QPushButton:pressed{
background-color:qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(50, 0, 143, 255), stop:1 rgba(39, 163, 122, 255))
}</string>
        </property>
        <property name="text">
         <string>Upload</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="cal">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{  
width: 30%;
 color: #fff;
  padding: 5px;
  font-size: 18px;
  text-transform: uppercase;
  border-radius: 10px;
border:none;
background: none;
}

QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(40, 0, 114, 255), stop:1 rgba(46, 191, 145, 255));
}
QPushButton:pressed{
background-color:qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(50, 0, 143, 255), stop:1 rgba(39, 163, 122, 255))
}</string>
        </property>
        <property name="text">
         <string>Calculate</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{  
width: 30%;
 color: #fff;
  padding: 5px;
  font-size: 18px;
  text-transform: uppercase;
  border-radius: 10px;
border:none;
background: none;
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(40, 0, 114, 255), stop:1 rgba(46, 191, 145, 255));
}
QPushButton:pressed{
background-color:qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(50, 0, 143, 255), stop:1 rgba(39, 163, 122, 255))
}</string>
        </property>
        <property name="text">
         <string>Table</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{  
width: 30%;
 color: #fff;
  padding: 5px;
  font-size: 18px;
  text-transform: uppercase;
  border-radius: 10px;
border:none;
}
QPushButton:hover{
	background-color: rgb(101, 64, 95);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="styleSheet">
         <string notr="true">
margin-left:0px;
border: none;
</string>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">font-size: 25px;
font: bold;
color: rgb(255, 255, 255);
</string>
            </property>
            <property name="text">
             <string>RESULTS</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <property name="styleSheet">
             <string notr="true">
QHeaderView
{
 Background:transparent; 
}
QHeaderView::section
{
 Font-size:14px;
 Font-family: &quot;Microsoft YaHei&quot;; 
 Color: #FFFFFF;
 
 Background:#60669B;
Border:none;
 Text-align:left; 
 
 
 Min-height: 49px;
    max-height:49px;             
 
 Margin-left:0px; 
 Padding-left: 0px;
}
 
QTableWidget
{
alternate-background-color: rgb(255, 222, 222) ;
 Background: #FFFFFF; 
 Border:none; 
 selection-background-color: qlineargradient(x1: 0, y1: 0, x2: 0.5, y2: 0.5,
                                stop: 0 #FF92BB, stop: 1 #60669b);
 Font-size: 12px; 
 Font-family:&quot;Microsoft YaHei&quot;; 
 Color:#666666; 
}
 
QTableWidget:: item 
{
 Border-bottom:1px solid #EEF1F7; 
}

 
QScrollBar::handle:vertical 
{
 Background: rgba(255,255,255,20%);  
 Border: 0px solid grey; 
 Border-radius: 3px; 
 Width: 8px; 
}
QScrollBar::vertical 
{
 Border-width: 1px; 
 Border-style: solid; 
 Border-color: rgba(255, 255, 255, 10%); 
 Width: 8px; 
 Margin:0px 0px 0px 0px; 
 Border-radius: 3px; 
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical 
{
 Background: rgba(255,255,255,10%); 
}
QScollBar::add-line:vertical, QScrollBar::sub-line:vertical 
{
 Background:transparent; background color of the top and bottom areas of the vertical scroll bar
}</string>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="gridStyle">
             <enum>Qt::SolidLine</enum>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QPushButton" name="pushButton_3">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	
	image: url(:/image/download.png);
    padding-top: 0px;
	margin-top:0px;
	background:none;

}
QPushButton:hover{
	background-color: rgb(47, 72, 99)
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTextBrowser" name="textBrowser">
            <property name="styleSheet">
             <string notr="true">
color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_3">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">font: 75 40pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background: none;</string>
            </property>
            <property name="text">
             <string>&lt;center&gt;WELCOME TO OUR GRADER&lt;/center&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_4">
         <property name="styleSheet">
          <string notr="true">
background:none;</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_2" native="true"/>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QProgressBar {
	font: 75 12pt &quot;MS Sans Serif&quot;;

border-radius: 10px;
background: rgb(116, 116, 116);

text-align:center;
color: rgb(200, 200, 200);
}
QProgressBar::chunk {
	background-color:  qlineargradient(x1: 0, y1: 0, x2: 0.5, y2: 0.5,
                                stop: 0 #FF92BB, stop: 1 #60669b);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(40, 0, 114, 255), stop:1 rgba(46, 191, 145, 255));
border-radius: 10px;
}</string>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 15pt &quot;MS Sans Serif&quot;;
color: rgb(255, 255, 255);
margin-left:50%;</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;LOADING..........&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_5">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">background:none;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>150</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">
font: 75 18pt &quot;MS Shell Dlg 2&quot;;

color: rgb(255, 255, 255);
background:none;</string>
            </property>
            <property name="text">
             <string>&lt;center&gt;NO FILE SELECTED&lt;/center&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="hehe.qrc"/>
 </resources>
 <connections/>
</ui>
